presentation/src/main/java/me/androidbox/presentation/NetworkResponseState.kt

```diff
@@ -0,0 +1,11 @@
+package me.androidbox.presentation
+
+sealed interface NetworkResponseState<out T> {
+    object Idle : NetworkResponseState<Nothing>
+
+    object Loading : NetworkResponseState<Nothing>
```

Comment:
I don't have a strong opinion on this, but these 2 are rather optional. You can also just set `isLoading` to true before you execute your network call and set it to false afterwards which is the approach I always use.